sesame/src/main/scala/org/w3/banana/sesame/SesameTransactor.scala

```diff
@@ -0,0 +1,28 @@
+package org.w3.banana.sesame
+
+import org.openrdf.repository.RepositoryConnection
+import org.w3.banana.{Transactor, SparqlUpdate, RDFStore}
+
+import scala.util.{Failure, Try}
+
+trait SesameTransactor extends Transactor[Sesame,RepositoryConnection] {
+
+  def r[T](conn: RepositoryConnection, body: => T): Try[T] = Try{
+    val result = body
+    conn.close()
```

Comment:
Here it is https://sourceforge.net/p/bigdata/discussion/676946/thread/14bdd6b1/

What is important:
```
Yes, bigdata has different transaction semantics from openrdf. Openrdf assumes that writers do not conflict and all transactions are read/write connections. Bigdata has read-only transactions that provide light weight snapshot isolation and allow you to read against any visible commit point. The highest write throughput is through the unisolated connection.
Read-only connection creation is very cheap in the standalone and HA replication cluster modes.
Do NOT hold open read-only connections. This will pin the commit point on which the connection reads and all subsequent commit points. That will cause the backing file to grow if you are concurrently writing on the database since it will not be able to release storage until you close those read-only transactions and un-pin the associated commit points.
```